frontend/web/components/pages/SegmentsPage.tsx

```diff
@@ -344,7 +344,7 @@ const SegmentsPage: FC<SegmentsPageType> = (props) => {
                       added to the segments based on the rules you create.{' '}
                       <Button
                         theme='text'
-                        href='https://docs.flagsmith.com/basic-features/managing-segments'
+                        href='https://docs.flagsmith.com/basic-features/segments'
```

Comment:
It might be worth adding this redirect in vercel.json, we'll probably be ranking for this url in search